go.mod

```diff
@@ -1,13 +1,15 @@
 module github.com/arduino/arduino-lint
 
-go 1.17
+go 1.22.0
+
+toolchain go1.22.3
```

Comment:
In some of my personal projects where I use Go 1.22, I had a lot of problems with the ["`check-config`" job](https://github.com/arduino/arduino-lint/blob/da69d84d6bf390b06e4052e5269525a8e9fb7f7a/.github/workflows/check-go-task.yml#L213) of the "Check Go" workflow failing on Dependabot PRs for Go module dependency bumps because Dependabot's PRs would change the `toolchain` directive to a whatever version of Go it uses, which was not the same as the version installed on the GitHub Actions runner by the **actions/setup-go** action. So I removed the `toolchain` directive from `go.mod` to avoid that.

Reply:
I've removed the `toolchain` directive from `go.mod`.